htlcswitch/test_utils.go

```diff
@@ -1056,14 +1056,14 @@ func serverOptionRejectHtlc(alice, bob, carol bool) serverOption {
 // createTwoClusterChannels creates lightning channels which are needed for
 // a 2 hop network cluster to be initialized.
 func createTwoClusterChannels(t *testing.T, aliceToBob,
-	bobToCarol btcutil.Amount) (*testLightningChannel,
+	bobToAlice btcutil.Amount) (*testLightningChannel,
```

Comment:
seems like this commit can be cherry-picked out of this PR and be merged into master first.

Reply:
I have reordered commits to put it at the beginning with the other cherry-pickable commit.